File: Volume 11 (1100 - 1199)/1115/1115.c
#include <stdio.h>
 
int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    while (a != 0 && b != 0) {
        if (a > 0 && b > 0) printf("primeiro\n");
        else if (a < 0 && b < 0) printf("terceiro\n");
        else if (a < 0 && b > 0) printf("segundo\n");
        else if (a > 0 && b < 0) printf("quarto\n");
        scanf("%d %d", &a, &b);
    }
    return 0;
}

File: Volume 11 (1100 - 1199)/1115/1115.py
l = [int(i)for i in input().split(' ')]
while l[0] != 0 and l[0] != 0:
    t = ""
    if l[0] > 0 and l[1] > 0: t = "primeiro"
    elif l[0] < 0 and l[1] < 0: t = "terceiro"
    elif l[0] < 0 and l[1] > 0: t = "segundo"
    elif l[0] > 0 and l[1] < 0: t = "quarto"
    print(t)
    l = [int(i)for i in input().split(' ')]
